package.json

```diff
@@ -59,8 +63,5 @@
     "publishchecks": "git diff --stat --exit-code HEAD && yarn build && node utils/pre-publish-check.js",
     "publish": "yarn run publishchecks && lerna publish",
     "test": "yarn jest"
-  },
-  "dependencies": {
-    "flow-enums-runtime": "^0.0.6"
```

Comment:
Shouldn't this still be in dependencies?

Reply:
It's in dev dependencies. The root package doesn't really have any prod dependencies - it just needs to be there at build time.